actor-profile.js

```diff
@@ -85,6 +85,7 @@ class ActorProfile extends HTMLElement {
     if (actorInfo.preferredUsername) {
       const pUserName = document.createElement('div')
```

Comment:
```suggestion
      const pUserName = document.createElement('a')
```

This needs to be an anchor if we want href to work.